omod/src/main/resources/config.xml

```diff
@@ -138,6 +138,13 @@
 	</messages>
 	<!-- /Internationalization -->
 
+	<globalProperty>
+		<property>registrationapp.paperRecordIdentifierDefinition</property>
+		<description>
+			Paper record identifier definition
```

Comment:
how about: Report Definition to retrieve the paper record identifier for a patient; default comes from paper record module